tests/integration_tests/sqla_models_tests.py

```diff
@@ -430,7 +425,7 @@ def test_multiple_sql_statements_raises_exception(self):
         }
 
         table = SqlaTable(
-            table_name="test_has_extra_cache_keys_table",
+            table_name="test_multiple_sql_statements",
```

Comment:
See my comment above. SQLite was complaining that said table already existed when I removed the "cleanup", though neither MySQL nor PostgreSQL did. I'm not entirely sure what's going on here, but it does seem like the table name was copypasta from the other test and thus the easiest solution was to rename the table in accordance with the test. 

I sense with this refactoring work it's a case of two steps forwards, one step back at time.